scripts/format-check.sh

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
 echo -e "[RUBOCOP] --> Init (wait a second)"
 
-if (bundle exec rubocop -x 2>/dev/null | grep 'no offenses detected' >/dev/null) ; then
+# For now, only check Layout cops
```

Comment:
For future reference: what's the issue with checking other cops?

Reply:
If I add the `Lint` cop for instance, I also need to fix the 54 offences that come with it, so I'd rather do it in a separate PR. Here I'm keeping the same functionality as before (`-x` is equivalent to `--only 'Layout' -a`).

Reply:
Makes sense 😄